spec/stepmod/converters/eqn_spec.rb

```diff
@@ -25,7 +25,7 @@
     XML
   end
   let(:output) do
-    "[[eqnGM1]]\n\n[stem]\n++++\nχ My node ms Italic text = V - E + 2F - L l - 2(S - G s) = 0\n++++\n\n"
+    "[[eqnGM1]]\n\n[stem]\n++++\nχ My node_{ms}Italic text = V - E + 2F - L_{l} - 2(S - G^{s})  = 0\n++++\n\n"
```

Comment:
Can we use heredoc/multi-line strings for comparison?

Reply:
Done